c++/include/orc/Reader.hh

```diff
@@ -180,6 +180,16 @@ namespace orc {
      */
     RowReaderOptions& forcedScaleOnHive11Decimal(int32_t forcedScale);
 
+    /**
+     * Set enable encoding block mode.
+     */
+    RowReaderOptions& setEnableLazyDecoding(bool enable);
```

Comment:
If this is for dictionary encoding, can we be more specific here? For example, setEnableLazyDictionaryDecoding?

Reply:
this is not only meant for dictionary decoding. This flag controls weather we use ColumnReader next() or nextEncoded() so I think we should leave it as is.